parlai/agents/transformer/modules.py

```diff
@@ -803,6 +841,21 @@ def _create_selfattn_mask(self, x):
         mask = mask.unsqueeze(0).expand(bsz, -1, -1)
         return mask
 
+    def reorder_incremental_state(self, incremental_state: dict, inds):
```

Comment:
Leave a type annotation for incr state and the others

Reply:
Good point - adding type annotations here